testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeColumnMappingMode.java

```diff
@@ -468,7 +373,7 @@ private void testSpecialCharacterColumnNamesWithColumnMappingMode(String mode)
                 " LOCATION 's3://" + bucketName + "/databricks-compatibility-test-" + tableName + "'" +
                 " TBLPROPERTIES (" +
                 " 'delta.columnMapping.mode' = '" + mode + "'," +
-                " 'checkpointInterval' = 3" +
+                " 'delta.checkpointInterval' = 3" +
```

Comment:
Interesting - Databricks supports both `checkpointInterval` and `delta.checkpointInterval` settings :).